docs/advanced-features/i18n-routing.md

```diff
@@ -0,0 +1,134 @@
+# Internationalized Routing
+
+<details>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/i18n-routing">i18n routing</a></li>
```

Comment:
Note: let's not forget to add this 😄

Reply:
Opened PR for this here https://github.com/vercel/next.js/pull/18206